products/compute/terraform.yaml

```diff
@@ -1200,6 +1200,44 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         diff_suppress_func: 'portDiffSuppress'
       sslHealthCheck: !ruby/object:Overrides::Terraform::PropertyOverride
         diff_suppress_func: 'portDiffSuppress'
+  RegionUrlMap: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_url_map_basic"
+        primary_resource_id: "regionurlmap"
+        vars:
+          region_url_map_name: "regionurlmap"
+          login_region_backend_service_name: "login"
+          home_region_backend_service_name: "home"
+          region_health_check_name: "health-check"
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: "map_id"
+      defaultService: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_expand: 'templates/terraform/custom_expand/url_map_only_set_string.go.erb'
+        description: The backend service or to use when none of the given rules match.
```

Comment:
was this supposed to be "or backend bucket"?

Reply:
Oops - backend buckets are not supported by RegionUrlMaps so it's supposed to be "" instead of "or"

Reply:
got it- in that case, do you also need the custom expands (since they claim to be for working around being able to support backend service and bucket in the same field)?

Reply:
Good point - I removed them.